src/faldbt/lib.py

```diff
@@ -258,6 +257,21 @@ def _write_relation(
     pdtable = pdsql.SQLTable(relation.identifier, pddb, data, index=False, dtype=dtype)
     alchemy_table: sqlalchemy.Table = pdtable.table.to_metadata(pdtable.pd_sql.meta)
 
+    # HACK: athena needs "location" property that is not passed by mock adapter
+    if adapter.type() == "athena":
+        s3_dir = adapter.config.credentials.s3_staging_dir
+        alchemy_table.dialect_options["awsathena"] = {
+            "location": f"{s3_dir}{alchemy_table.schema}/{alchemy_table.name}/",
+            "tblproperties": None,
+            "compression": None,
+            "bucket_count": None,
+            "row_format": None,
+            "serdeproperties": None,
+            "file_format": None,
+            "partition": None,
+            "cluster": None,
+        }
```

Comment:
instead of setting all these `None`, can you do `alchemy_table.dialect_options["awsathena"]["location"] = ...` or is `alchemy_table.dialect_options["awsathena"]` not set?

Reply:
These are checked one by one by PyAthena but without using a getter, so we need to define them.

Reply:
Yes, but I meant that maybe the already exist in the `alchemy_table.dialect_options["awsathena"]` object and just set the specific `location` property.